sig-architecture/api-review-process.md

```diff
@@ -0,0 +1,247 @@
+
+To qualify, aspiring approvers should reference at least 5 provisional reviews of new API types and 10 provisional reviews of changes to existing API types with no substantive disagreement or additions by approver. Aspiring approvers must be able to demonstrate competence both for new API review and for compatibility concerns with modifications to existing APIs.
+
+An indication of whether a provisional review was seen as qualifying should be part of the critique/coaching feedback given ("your provisional review caught everything important that I saw, +1")
+
+### Training Reviews
+
+For those wishing to become formal reviewers, the process has an alternate path where API reviews that are not time-sensitive can be initially reviewed, and then collaboratively examined with an approver/mentor. This can be accomplished asynchronously as comments in the review document, or ideally by pairing on the approver examination. 
+
+Once the provisional review has been accepted by the approver, it then goes to the submitters just as it would in the normal process. 
+
+As aspiring reviewers gain proficiency, they may pair with an approver on time-sensitive reviews to better understand the process with less impact on the review duration. 
+
+Aspiring reviewers should reach out the the moderator on slack.  The moderator will add them to the list of aspiring reviewers and assign training reviews as they become available. 
+
+## Aspirations for future iterations of this process
+
+* Make a video on this process as guidance for contributors
+
+* Review the API conventions doc for freshness and organization
+
+* Separate and keep separate the process and the API conventions parts of all our docs.  That way, people who are not subject to our process can read a more succinct set of docs on how to make good APIs.
+
+* Consider a new home for our API docs that is prettier than Github-flavored markdown ( a new tab on kubernetes.io or a new site) 
+
+* Given the level of completeness of the conventions this process should generate documentation of undocumented conventions, which might be pre-existing, or might be new.
+
+* Ideally this process should lend itself to some automation. It currently does not. [ TBD: Create an umbrella issue for this ]
+
+    * [We already tag PRs with api-change](https://github.com/kubernetes/test-infra/blob/1611f3be80713f9df933e25e4cf9a2b538302ef5/mungegithub/misc-mungers/deployment/kubernetes/path-label.txt#L7-L17)
```

Comment:
I want to get rid of mungegithub.  This functionality is supported in prow today, it's just that nobody has setup an OWNERS file to implement it.  We have (probably undocumented) support for regexes and labeling to accomplish this in one file.